src/integrationTest/sam.test.ts

```diff
@@ -225,7 +223,8 @@ describe('SAM Integration Tests', async () => {
                     console.log(`subSuiteTestLocation: ${subSuiteTestLocation}`)
 
                     await createSamApplication(subSuiteTestLocation)
-                    const appPath = path.join(subSuiteTestLocation, samApplicationName, scenario.path)
+                    // TODO: useful?
```

Comment:
`appPath` is the file that contains the app source code for the tests scoped to the suite on line 210. It appears to be used on line 329.